Started by timer
Running as SYSTEM
Building in workspace /var/root/.jenkins/workspace/hot_user
using credential root
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url git@github.com:kirinisland/zaoci-tool.git # timeout=10
Fetching upstream changes from git@github.com:kirinisland/zaoci-tool.git
 > git --version # timeout=10
using GIT_SSH to set credentials 
 > git fetch --tags --force --progress -- git@github.com:kirinisland/zaoci-tool.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 67503715f5806fd7cdba1c9803cb59378dd4d097 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 67503715f5806fd7cdba1c9803cb59378dd4d097 # timeout=10
Commit message: "added UserIdentity table"
 > git rev-list --no-walk 67503715f5806fd7cdba1c9803cb59378dd4d097 # timeout=10
[hot_user] $ /bin/sh -xe /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/jenkins4366847624660336176.sh
+ python3 jobs/hot_user.py --count 25
Hot users after date: 2020-11-09 00:00:00
点赞最多: https://api2.bmob.cn/1/classes/EverydayLyric?&limit=25&where={"validStatus": {"$gte": 2, "$lte": 3}, "createdAt": {"$gte": {"__type": "Date", "iso": "2020-11-09 00:00:00"}}}&keys=user,likeCount,commentCount,bookmarkCount&order=-likeCount
Traceback (most recent call last):
  File "jobs/hot_user.py", line 170, in <module>
    main()
  File "jobs/hot_user.py", line 74, in main
    like_most_users = fetch_hot_users(like_url, headers)
  File "jobs/hot_user.py", line 116, in fetch_hot_users
    for user in raw.json().get("results"):
  File "/Library/Python/3.7/site-packages/requests/models.py", line 897, in json
    return complexjson.loads(self.text, **kwargs)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Build step 'Execute shell' marked build as failure
Finished: FAILURE
